arktype.io/src/components/features.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-lines-per-function */
+import { Box, Stack, SvgIcon, Typography } from "@mui/material"
+import One from "@site/static/img/features/1.svg"
+import Two from "@site/static/img/features/2.svg"
+import Three from "@site/static/img/features/3.svg"
+import Four from "@site/static/img/features/4.svg"
+import React from "react"
+import details from "../featuresConfig.json"
+
+const features = {
```

Comment:
It's hard to imagine an object with keys `One`, `Two` etc... could not somehow be converted to an array unless we are actually trying to display the number names on the page somewhere.